frontend/package.json

```diff
@@ -50,9 +52,11 @@
     "@vitejs/plugin-react-swc": "^3.5.0",
     "babel-jest": "^29.7.0",
     "dotenv": "^16.4.5",
-    "lefthook": "^1.7.15",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
+    "jest-fixed-jsdom": "^0.0.2",
```

Comment:
We shouldn't have to add this to get jest working in the repository.

Reply:
Wasn't working any other way

Reply:
This one is a bit tricky, as it's more about integrating MSW with Jest rather than just Jest alone. The jest-fixed-jsdom package is recommended by the MSW developers, but we don’t necessarily need to use it since there are plenty of solutions available online, and we can implement a workaround ourselves. Jake, if you have a simpler solution, please feel free to share.

Reply:
Maybe we should consider switching to Vitest instead of Jest. It's modern, integrates well with Vite (which we're already using), is faster, and has built-in TypeScript support.

Reply:
Lets just keep it and move on. Once this is working we shouldn't have to touch the configuration.